tests/test_analysis/test_feature_relevance/test_relevance.py

```diff
@@ -0,0 +1,8 @@
+from etna.analysis.feature_relevance import StatisticsRelevanceTable
+
+
+def test_statistics_relevance_table(simple_df_relevance):
```

Comment:
Could you please fix the fixture typing in confest

Reply:
done